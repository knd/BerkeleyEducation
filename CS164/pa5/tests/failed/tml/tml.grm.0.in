<html>
    <head class="what" class="bar biz" what="who" when="then">
        <title>Foo What</title>
    </head>
    <body>
        Who's on first?<br />
        No, <b>he's</b>
        <p></p>
    </body>
</html>