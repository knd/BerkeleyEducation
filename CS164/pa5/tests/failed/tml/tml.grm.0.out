{'name': 'html', 'children': {0: {'children': {0: {'name': 'title', 'children': {0: {'text': 'Foo What', 'name': 'Word', 'children': {}}}}}, 'what': 'who', 'when': 'then', 'class': 'bar biz', 'name': 'head'}, 1: {'name': 'body', 'children': {0: {'text': "Who's on first?", 'name': 'Word', 'children': {}}, 1: {'name': 'br', 'children': {}}, 2: {'text': 'No,', 'name': 'Word', 'children': {}}, 3: {'name': 'b', 'children': {0: {'text': "he's", 'name': 'Word', 'children': {}}}}, 4: {'name': 'p', 'children': {}}}}}}
